src/test/java/co/cask/wrangler/steps/PipelineTest.java

```diff
@@ -93,4 +95,26 @@ public void testSplitWithNull() throws Exception {
     Assert.assertEquals("1,2,a,A", row.getValue("firstCol"));
     Assert.assertNull(row.getValue("secondCol"));
   }
+
+  @Test
+  public void testEscapedStrings() throws Exception {
+    List<Step> steps = new ArrayList<>();
+    Row row = new Row("__col", StringEscapeUtils.unescapeJava("1\\ta"));
```

Comment:
So in cdap, the Wrangler Spec has the actual characters '\' and 't' in it, but the data coming through has the "tab" character in it. This test is mimicking that by unescaping the \\t into an actual "tab" character.